AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -37,6 +37,7 @@ aura.top#?#.sidebar-nav > li.sidebar-nav-item-small-link:has(> a[href^="https://
 !#endif
 !
 indianexpress.com,financialexpress.com##.app-btn
+duolingo.com##._1e7JD
```

Comment:
cannot find it. Changed?

Reply:
<details>
<summary>No</summary>
<img src="https://user-images.githubusercontent.com/18536328/89108140-c9632400-d43e-11ea-8f2a-8d5662923ba2.png" />
</details>